kserve.advisories.yaml

```diff
@@ -193,3 +197,8 @@ advisories:
             componentType: python
             componentLocation: /usr/lib/python3.11/site-packages/starlette-0.36.3.dist-info/METADATA, /usr/lib/python3.11/site-packages/starlette-0.36.3.dist-info/RECORD
             scanner: grype
+      - timestamp: 2024-10-25T16:08:42Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-in-execution-path
+          note: vulnerable version of Starlette python package used (v0.36.3, fixed is v0.40.0), but no vulnerable functions are used in kserve. The vulnerable DoS attack vector involves a very specific case of functions in the starlette.applications and starlette.routing subpackages, but those are never used in kserve. We cannot simply upgrade to the fixed version because kserve enforces starlette version >=0.37.2,<0.39.0. Waiting on upstream to migrate to fixed versions, but in the meantime, false-positive-determination."
```

Comment:
is there any upstream link we ca attach in the notes?

Reply:
Not that I can find, this determination came from my manual inspection of the kserve code at v0.13.1.